NitroxServer/GameLogic/Entities/Spawning/DeterministicBatchGenerator.cs

```diff
@@ -26,8 +28,17 @@ public NitroxId NextId()
         {
             byte[] bytes = new byte[16];
             random.NextBytes(bytes);
+            NitroxId id = new NitroxId(bytes);
 
-            return new NitroxId(bytes);
+            if (nitroxIds.Contains(id))
+            {
+                NitroxId newId = NextId();
+                nitroxIds.Add(newId);
+                return newId;
+            }
```

Comment:
What is this being used for?

Reply:
ensuring the IDs are always different no matter what

Reply:
I think we should remove.   This should not happen.